column_transforms/datepart/datepart.yaml

```diff
@@ -1,19 +1,20 @@
 name: datepart
-description: Extracts a specific part of a date column. For example, if the input is '2021-01-01', you can ask for the year and get back 2021.
+description: |
+  Extracts a specific part of a date column. For example, if the input is '2021-01-01', you can ask for the year and get back 2021.
+  
+  An exhaustive list of valid date parts can be [found here](https://docs.snowflake.com/en/sql-reference/functions-date-time.html#label-supported-date-time-parts).
 arguments:
-  date_columns:
-    type: column_list
-    description: names of column(s) you want to date-part
-  date_part:
-    type: date_part
-    description: the desired grain of the date; an exhaustive list of valid date parts can be [found here](https://docs.snowflake.com/en/sql-reference/functions-date-time.html#label-supported-date-time-parts).
+  dates:
+    type: datepart_dict
```

Comment:
This type `datepart_dict` is not yet supported on the UI side.   Should we add it?